src/style.css

```diff
@@ -0,0 +1,26 @@
+@import "./blocks/product-card.css";
+@import "./blocks/stars.css";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+body,
+p,
+h1,
+h2,
+h3,
+h4,
+ul,
+span,
+img {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Move this stile to another file `normalize.css`, and import it on the top of file